api-extractor/gulpfile.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const build = require('@microsoft/node-library-build');
+
+const tslintCommon = require('@microsoft/sp-tslint-rules');
```

Comment:
Not sure where you are seeing that lint file. The default lint rules that actually are used by `node-library-build` are in `gulp-core-build-typescript/src/defaultTsLint.json`

I enabled it there, but for now this package has a cyclic dependency and therefore it builds using the last published version of the repo, so the rule will be disable until we republish.